test-apps/ui-test-app/src/frontend/appui/LocalFileSupport.ts

```diff
@@ -12,22 +12,26 @@ import { IModelError } from "@bentley/imodeljs-common";
 export class LocalFileSupport {
 
   public static localFilesSupported = (): boolean => {
-    if (!SampleAppIModelApp.testAppConfiguration?.snapshotPath) {
+    if (ProcessDetector.isElectronAppFrontend)

```

Comment:
please use `ElectronApp.isValid` instead of ProcessDetector

Reply:
Submitted PR https://github.com/imodeljs/imodeljs/pull/941